src/function.coffee

```diff
@@ -3,15 +3,17 @@ store = require('./store')
 calls = require('./store/calls')
 stubbings = require('./store/stubbings')
 
-module.exports = (name) ->
-  _.tap createTestDoubleFunction(), (testDouble) ->
+module.exports = (name, options = {}) ->
+  _.tap createTestDoubleFunction(name, options), (testDouble) ->
     if name?
       store.for(testDouble).name = name
       testDouble.toString = -> "[test double for \"#{name}\"]"
     else
       testDouble.toString = -> "[test double (unnamed)]"
 
-createTestDoubleFunction = ->
+createTestDoubleFunction = (name, options)->
   testDouble = (args...) ->
     calls.log(testDouble, args, this)
+    if options.log
+      console.log("Test Double Function '%s' called with: ", name, args)
```

Comment:
Style points, would you mind taking advantage of interpolation here?

`console.log("Test Double Function '#{name}' called with: ", args)`